lib/src/firebase_storage_mocks_base.dart

```diff
@@ -16,6 +17,26 @@ class MockFirebaseStorage extends Mock implements FirebaseStorage {
     return MockReference(this, path);
   }
 
+  @override
+  Reference refFromURL(String url) {
```

Comment:
Same here. If you copied this from another repo, please include the link. It seems like it came from https://github.com/firebase/flutterfire/blob/3dfc0997050ee4351207c355b2c22b46885f971f/packages/firebase_storage/firebase_storage/lib/src/firebase_storage.dart#L111.